EIPS/eip-6969.md

```diff
@@ -0,0 +1,126 @@
+---
+eip: 6969
```

Comment:
```suggestion
eip: 6968
```
Manually assigning 6968 as there isn't a specification ready yet. 
 See: https://github.com/ethereum/EIPs/pull/6969#issuecomment-1530710749

Reply:
Please rename the file to match the assigned EIP # EIP6969.